### ISSUE TITLE
Finding keys with non unique labels

### ISSUE THREAD

**Issue by bernard-wagner** [eclipse-keypont/crypto11#50]
The find functions assume only one result will be returned when querying a label, however, labels are not always unique. 

As an example, it is possible that more than one keypair exists with the label "testing", but without knowing the ID you will always only get the first entry.
```
signer, err := crypto11.FindKeyPair(nil, []byte("testing")) 
```

Suggestion would be to either change the function to always return a slice, or to add functions that return a slice. 
```
FindKeyPair(id []byte, label []byte) ([]Signer, error)
FindKeyPairs(label []byte) ([]Signer, error)
```

**Comment by dmjones** [comment]
Good point. We can't change the return value for existing functions, so adding the overload of `FindKeyPairs` is a better choice.

**Comment by dmjones** [comment]
Fixed in #57.
